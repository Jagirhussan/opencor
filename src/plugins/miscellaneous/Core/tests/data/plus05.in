<!--
a=b/c+d/e;
-->
<math xmlns="http://www.w3.org/1998/Math/MathML">
    <apply>
        <eq/>
        <ci>a</ci>
        <apply>
            <plus/>
            <apply>
                <divide/>
                <ci>b</ci>
                <ci>c</ci>
            </apply>
            <apply>
                <divide/>
                <ci>d</ci>
                <exponentiale/>
            </apply>
        </apply>
    </apply>
</math>
