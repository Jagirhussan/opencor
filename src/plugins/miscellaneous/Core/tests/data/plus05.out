<math xmlns="http://www.w3.org/1998/Math/MathML">
    <mrow>
        <mrow>
            <mi mathvariant="italic">a</mi>
        </mrow>
        <mo>=</mo>
        <mrow>
            <mfrac>
                <mrow>
                    <divide/>
                    <mrow>
                        <mi mathvariant="italic">b</mi>
                    </mrow>
                </mrow>
                <mrow>
                    <mrow>
                        <mi mathvariant="italic">c</mi>
                    </mrow>
                </mrow>
            </mfrac>
            <mo>+</mo>
            <mfrac>
                <mrow>
                    <divide/>
                    <mrow>
                        <mi mathvariant="italic">d</mi>
                    </mrow>
                </mrow>
                <mrow>
                    <mi>e</mi>
                </mrow>
            </mfrac>
        </mrow>
    </mrow>
</math>
